app/scripts/controllers/permissions/restrictedMethods.js

```diff
@@ -146,7 +146,7 @@ function getExternalRestrictedMethods (permissionsController) {
           // Here is where we would invoke the message on that plugin iff possible.
           const handler = permissionsController.pluginsController.rpcMessageHandlers.get(origin)
           if (!handler) {
-            res.error = ethErrors.methodNotFound({
+            res.error = ethErrors.rpc.methodNotFound({
```

Comment:
I've been catching these missing `.rpc` instances around the code, too. Just so I know, what was the motivation to change this API?

I'm pretty sure if this was a mono-repo (rpc errors and our other repos), we could use IDE tools like "refactor" to make sure we're changing all uses of a method everywhere at once, but since we don't do that, changing APIs is still scary and kinda dangerous if not done methodically.

Reply:
I made the change a long time ago for `eth-json-rpc-errors@2.0.0`, because I thought mixing the kind of things we found in a particular level of the API namespace (in `<2.0.0`, i.e. JSON RPC errors directly under `rpcErrors` as well as the `eth` property, which had all the Ethereum Provider errors).

When I upgraded the module I made the changes everywhere, but I didn't communicate the change to you and others, and I think that's why we're finding these errors.